README.md

```diff
@@ -26,10 +26,13 @@ Refer to the README files in each subdirectory of this repo for specific instruc
 * [TypeScript](http://www.typescriptlang.org/)
 
 ## Contributing
-Anyone and everyone is welcome to contribute. We accept pull requests. Feel free to:  
-* Report issues
-* Contribute code
-* Improve documentation
+
+Esri welcomes contributions from anyone and everyone. Please see our [guidelines for contributing](https://github.com/esri/contributing).
+
+We use issues and pull requests to address problems found in the [ArcGIS API for Javascript](http://js.arcgis.com/) here.  If you're asking for help please utilize the supplementary resources below.
```

Comment:
https://github.com/Esri/jsapi-resources/issues/33 is an example of a situation in which someone proposed a fix for a problem in the API in this repo.

in general, you're right. distinguishing exactly what is being tracked here and what types of contributions are the most appropriate and guiding users to alternative (and superior resources) for other help is what's needed.